src/components/footer/Footer.js

```diff
@@ -1,22 +1,67 @@
 import React from 'react';
-import { translate } from 'react-i18next';
+import { translate, Interpolate } from 'react-i18next';
 import 'i18n/i18n';
+import Panel from 'components/panel/Panel';
+import Button from 'components/button/Button';
 import styles from './footer.scss';
+import constant from 'config/constants';
 
-const Footer = ({ t }) => (
-  <footer className={styles.footerMain}>
-    <div className={styles.footerInner}>
-      <a href="#terms" className={styles.footerLink}>
-        {t('footer-terms')}
+const Footer = ({ t }) => {
```

Comment:
To help readability, could you use a class instead? There are several functions inside, I think a class would be easier to read

Reply:
I don't really like using classes for presentational component, may be I can move all the functions out of the component, and since its ES6 modules the variables will not leak out of the module. What say?

Reply:
Yes, please do that.

Reply:
why not use a simple `function Footer({ t })`?

Nothing is going to change inside, so it's better than a `function expression` isn't it? In this case, I prefer: `function` > `stateless` > `statefull`

Reply:
@tureey What is the difference between `function` and `stateless` components?